commercetools/resource_store.go

```diff
@@ -48,6 +48,13 @@ func resourceStore() *schema.Resource {
 				Optional:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
 			},
+			"countries": {
+				Description: "A two-digit country code as per " +
+				"[ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)",
+				Type:        schema.TypeList,
+				Optional:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
```

Comment:
My preference would be that in this case we stick as close to the HTTP API definition as possible; so instead of a list of strings that we have a `StoreCountry` nested object with a CountryCode field. This way if the object gets expanded in the future we don't have issues adding this to the terraform provider.

So that would mean a terraform config like the following:
```terraform
resource "commercetools_store" "my-store" {
  ...
  
  countries = [
    {country_code = 'nl'}
  ]
}

```

Reply:
And adding some validation on the nested `country_code` field would also be great! That way we can prevent any issues in the data before executing the plan